script.js

```diff
@@ -0,0 +1,131 @@
+document.getElementById('btnAdd').addEventListener('click', receiveText);
+document.getElementById('todoText').addEventListener('keydown',function(e){
+    if(e.key === 'Enter'){
+        receiveText();
+    }
+})
+document.getElementById('reloadAll_td').addEventListener('click', reloadTodo);
+document.getElementById('reloadAll_dn').addEventListener('click', reloadDone);
+
+reloadSum();
+
+//텍스트를 받아 삭제 버튼을 포함한 테이블의 한 행을 만들어주는 함수
+function addList(todoStr){
+    var tr=document.createElement('tr');
+    var td_txt=document.createElement('td');
+    td_txt.setAttribute('class', 'td_txt');
+    td_txt.innerHTML = todoStr;
+    tr.appendChild(td_txt);
+
+    var td_dlt=document.createElement('td');
+    td_dlt.setAttribute('class', 'td_dlt');
+    var input_dlt=document.createElement('input');
+    input_dlt.setAttribute('type', 'button');
+    input_dlt.setAttribute('value', "X");
+    input_dlt.setAttribute('class', 'btn_dlt');
+    input_dlt.onclick=dltList;
+    td_dlt.appendChild(input_dlt);
+    tr.appendChild(td_dlt);
+    return tr;
+}
+
+//입력창에서 텍스트만 받아내어 todoList()로 넘겨주는 함수
+function receiveText(){
+    var todoText = document.querySelector("#todoText");
+    if(!todoText.value){
```

Comment:
value의 값을 이렇게 체크하게 되면 스페이스바로 공백을 입력한 경우에는 그대로 To-Do에 텍스트가 추가되게 됩니다! 이럴 때 정규식 `trim()`을 사용한다면 공백만 입력하는 경우도 체크를 할 수 있답니당
```suggestion
    if(!todoText.value.trim()){
```